features.yaml

```diff
@@ -4,28 +4,255 @@ features:
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Trace Sampling"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Request Classification"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Observability
+  - name: "WebAssembly Extension"
+    level:
+     - checklist:
+       maturity: Alpha
+       nextExpectedPromotion:
+       area: Extensibility
+  - name: "Service-to-service Mutual TLS"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Kubernetes: Service Credential Distribution"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Certificate management on Ingress Gateway"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Pluggable Key/Cert Support for istio CA"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Authorization"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "End User (JWT) Authentication"
     level:
      - checklist:
        maturity: Beta
        nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Automatic mutual TLS"
+    level:
+     - checklist: features/auto_mtls.md
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "VM: Service Credential Distribution"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+  - name: "Mutual TLS Migration"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Standalone Operator"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Core
+  - name: "Kubernetes: Envoy Installation and Traffic Interception"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Core
+  - name: "Kubernetes: Istio Control Plane Installation"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Core
+  - name: "Multicluster Mesh"
+    level:
+     - checklist: features/Multi-cluster support.md
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Core
+  - name: "External Control Plane"
+    level:
+     - checklist: features/external_istiod.md
+       maturity: Alpha
+       nextExpectedPromotion:
+       area: Core
+  - name: "Kubernetes: Istio Control Plane Upgrade"
```

Comment:
in place or revision?

Reply:
I think that refers to in-place upgrades because I think that revision based are alpha. Confirming though.

Reply:
@howardjohn would you be able to comment here? The Istio feature status page isn't clear to me.